politeiad/politeiad.go

```diff
@@ -1072,9 +1072,14 @@ func _main() error {
 	}()
 
 	log.Infof("Version : %v", version.String())
+	log.Infof("Github Version: %v", version.GithubVersion())
 	log.Infof("Network : %v", activeNetParams.Params.Name)
 	log.Infof("Home dir: %v", loadedCfg.HomeDir)
 
+	if version.GithubVersion() == "(devel)" && !loadedCfg.Devel {
+		log.Warnf("Warning: no build information available")
```

Comment:
If it is preferred, we can make pi panic here instead of issuing a warning.

Reply:
If we're going to make it a panic then it makes sense to have a --devel override flag.  If we're going to only make it a warning then you don't really need the --devel override flag.  I'm fine with keeping it a warning.

Reply:
I think issuing a warning is the best option for us as well. Removed the --devel flag on the last commit :+1: